extensions/catalogs/catalog-content-resolution.adoc

```diff
@@ -0,0 +1,25 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="catalog-content-resolution"]
+= Catalog content resolution
+include::_attributes/common-attributes.adoc[]
+:context: catalog-content-resolution
+
+toc::[]
+
+When you specify the cluster extension you want to install in a custom resource (CR), {olmv1-first} uses catalog selection to resolve what content is installed.
+
+You can perform the following actions to control the selection of catalog content:
+
+* Specify metadata to select the catalog by name or labels
```

Comment:
Technically, we don't support name directly. We create a label with the name that can then be selected upon

Reply:
@tmshort The assumption is that the name label would be available by-default with cluster extensions in a catalog?

Reply:
Addressed: https://github.com/openshift/openshift-docs/pull/88394/commits/341b48ef8fe228c72f1fd1fd70bf11bcc76699d3